app/views/obituaries/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="obituary-image">
```

Comment:
This still needs to be styled. I just added the bare minimum to get it to pass the tests.